conftest.py

```
@@ -724,24 +724,20 @@ def plan(
-            "name": pod.name,
-            "namespace": pod.namespace,
-            "module": pod.__module__,
-        })
+        try:
+            vm_obj = VirtualMachine(
+                client=ocp_admin_client,
+                name=vm["name"],
+                namespace=target_namespace,
+            )
+            if vm_obj.exists:
+                LOGGER.info(f"Deleting VM {vm_obj.name} after test")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)